_drafts/community-group-overview.md

```diff
@@ -1,48 +1,55 @@
 ---
 title: "Overview of Community Groups"
+author: The Coop Team
 categories:
   - Community
 tags:
+  - user groups
 ---
 
 #### Overview of Community Groups
 
-The [Bioinformatics & Data Science Cooperative](https://research.fhcrc.org/coop/en.html), otherwise known as The Coop, is a cooperative that includes all groups and individuals at [Fred Hutch](https://www.fredhutch.org/en.html) interested in bioinformatics and data science. To help stimulate that interest and encourage learning and growth the Coop has a [newsletter](https://research.fhcrc.org/coop/en/newsletter.html), [slack channels](_posts/_drafts/DRAFT_2019-11-07-post-Slack.md), and a [calendar](https://fredhutch.github.io/FHBig/calendar/) of events. The Coop has also established and actively supports and encourages [community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html). Community groups are loose associations of individuals in the community who come together regularly around a particular tool, language, skill etc. and meet either quarterly, monthly, or weekly depending on the groups needs. These meetings usually focus on best practices, new approaches, tools, information sharing, or demoing a new workflow or process that others in the community may find valuable.
+To help stimulate interest in bioinformatics and data science while also encouraging learning and growth we here at [the Coop](https://research.fhcrc.org/coop/en.html) have started a [newsletter](https://research.fhcrc.org/coop/en/newsletter.html), manage [slack channels](https://fredhutch.github.io/coop/community/slack-new/), and publish a [calendar](https://fredhutch.github.io/FHBig/calendar/) of events. We've also established and actively support and encourage community groups. [Community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html) are loose associations of individuals in the community who come together regularly around a particular tool, language, skill etc. and meet either quarterly, monthly, or weekly depending on the groups needs. These meetings usually focus on best practices, new approaches, tools, information sharing, or demoing a new workflow or process that others in the community may find valuable.
 
-#### What they are
+#### Current Community Groups
 
-Here are some of the current community groups that we have here at the Hutch but it's not exhaustive as you can always create a new one but following the [guidelines here](.../2010-01-07-post-overview_community_group#how-to-start-one).
+Here are some of the [current community groups](https://research.fhcrc.org/coop/en/community/hosted-groups.html) that we have here at the Hutch but it's not exhaustive as you can always create a new one by following the guidelines below.
```

Comment:
I would soften the language here: say, "Interested in starting a community group focused on a topic not listed here? See below for instructions on how to request a new group."

Reply:
good idea....I incorporated your wording